### DISCUSSION TITLE
Snapshot selection with Frigate+ License Plate attribute

### DISCUSSION THREAD

**Discussion by gadget-man** [blakeblackshear/frigate#9121] [General]
I've got Frigate+ detecting vehicles with number plates. However, I'm finding that the snapshot that is saved by Frigate tends to be a bit too late in the clip, so it's failing to select the best shot. For example the attached is the snapshot of a vehicle driving into our drive. The snapshot is right at the last minute and so the plate is very blurred, but if the snapshot had been taken a couple of hundred milliseconds earlier, it would have been much clearer. Is there any way we can adjust this in the settings?
![snapshot-1703864267 351188-b1si7e](https://github.com/blakeblackshear/frigate/assets/24406912/7d2e5374-24ab-4692-b139-40fd289c0758)

**Comment by NickM-27** [comment]
There is no way to adjust the behavior currently, it is based on the area / score of the license plate

**Comment by gadget-man** [comment]
Ah ok. Is there a way I could use a mask of some sort to force it to ignore the plate when it’s clearly too close to the edge of the frame?

**Reply by NickM-27** [reply]
the best way to do that if you wanted to would be to set a max_area filter for license_plate
